README.md

```diff
@@ -90,13 +90,13 @@ The agent works with Java runtimes version 8 and higher. For the full list of re
 
 ## Get started
 
-Follow these steps to automatically instrument your application using the Java (or JVM) agent.
+Follow these steps to automatically instrument your application using the Java (or JVM) agent:
```

Comment:
`Java (or JVM) agent` is a bit weird, makes it looks like there are 2 different agents

Reply:
Changed

Reply:
Seems like we are using both JVM agent and Java agent in this doc, perhaps could pick one and use it everywhere